### DISCUSSION TITLE
Collection based webhooks

### DISCUSSION THREAD

**Discussion by ChinaeduO** [outline/outline#6318] [Ideas]
As of my understanding, it is currently only possible to add webhooks that trigger on events happening in any collection. It would be nice to have webhooks binded to a collection.

**Comment by tommoor** [comment]
While it's easy to filter on the receiving end it would be nice to reduce outgoing data in this way. Would accept a PR for this functionality, but right now the maintaining team isn't prioritizing changes to webhooks.
